testing/mock/mock.go

```diff
@@ -35,6 +35,7 @@ const (
 var (
 	MockAcknowledgement             = channeltypes.NewResultAcknowledgement([]byte("mock acknowledgement"))
 	MockFailAcknowledgement         = channeltypes.NewErrorAcknowledgement(fmt.Errorf("mock failed acknowledgement"))
+	ErrorMock                       = fmt.Errorf("mock failed acknowledgement")
```

Comment:
On channel upgrades, we added an error to test application callbacks with. Maybe I can upstream this change and we can use that? see https://github.com/cosmos/ibc-go/blob/04-channel-upgrades/testing/mock/mock.go#L42

Reply:
see #4189

Reply:
using this error now